src/main/java/com/laserfiche/api/client/httphandlers/UsernamePasswordHandler.java

```diff
@@ -1,19 +1,19 @@
 package com.laserfiche.api.client.httphandlers;
 
-import com.laserfiche.api.client.model.CreateConnectionRequest;
-import com.laserfiche.api.client.model.SessionKeyInfo;
 import com.laserfiche.api.client.apiserver.TokenClient;
 import com.laserfiche.api.client.apiserver.TokenClientImpl;
+import com.laserfiche.api.client.model.CreateConnectionRequest;
+import com.laserfiche.api.client.model.SessionKeyInfo;
 
 import java.util.concurrent.CompletableFuture;
 
 public class UsernamePasswordHandler implements HttpRequestHandler {
     private String accessToken;
-    private final String GRANTTYPE = "password";
-    private String repositoryId;
-    private String baseUrl;
-    private TokenClient client;
-    private CreateConnectionRequest request;
+    private final String GRANT_TYPE = "password";
+    private final String REPOSITORYID;
```

Comment:
Please don't upper case all `final` variables because: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
ok changed it back all to lower case